ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -6,9 +6,9 @@
     "PageDisplayOrder": 5,
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
-    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\" OR respondent1ClaimResponseTypeForSpec=\"FULL_ADMISSION\"",
+    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\"",
```

Comment:
Changing the conditions as this means that in full admission the respondent cannot specify if they have already paid or otherwise.